testgres/pubsub.py

```diff
@@ -0,0 +1,200 @@
+# coding: utf-8
+"""
+Unlike physical replication the logical replication allows users replicate only
+specified databases and tables. It uses publish-subscribe model with possibly
+multiple publishers and multiple subscribers. When initializing publisher's
+node ``allow_logical=True`` should be passed to the :meth:`.PostgresNode.init()`
+method to enable PostgreSQL to write extra information to the WAL needed by
+logical replication.
+
+To replicate table ``X`` from node A to node B the same table structure should
+be defined on the subscriber's node as logical replication don't replicate DDL.
+After that :meth:`~.PostgresNode.publish()` and :meth:`~.PostgresNode.subscribe()`
+methods may be used to setup replication. Example:
+
+>>> from .api import get_new_node
```

Comment:
This should be changed to `from testgres import get_new_node`. Documentation should contain working code, even it would require remove doctest.

Reply:
Agree